src/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -269,11 +328,115 @@ const handleOpenAuth = (mediator, artist) => {
   })
 }
 
+const RoundedFlex = styled(Flex)`
+  border-radius: 100px;
+`
+
+const renderAuctionHighlight = artist => {
+  if (
+    !artist.auctionResultsConnection ||
+    artist.auctionResultsConnection.edges.length < 1
+  ) {
+    return null
+  }
+  const topAuctionResult =
+    artist.auctionResultsConnection.edges[0].node.price_realized.display
+
+  return (
+    <RoundedFlex
+      background={color("black5")}
+      width="auto"
+      py={5}
+      px={10}
+      mr={5}
+    >
+      <AuctionIcon pr={5} />
+      <Sans size="2">{topAuctionResult} Auction Record</Sans>
+    </RoundedFlex>
+  )
+}
+
+const CATEGORIES = {
+  "blue-chip": "Blue chip",
```

Comment:
This looks like it's replicated in [another file](https://github.com/artsy/reaction/blob/29703cd9e891b85b9f13b5907528a3e900bb328c/src/Components/v2/SelectedCareerAchievements.tsx#L19). That's only 2 places but you could easily just export that constant and use it here.